Tests/SwiftProtobufTests/Test_Map.swift

```diff
@@ -68,6 +68,7 @@ class Test_Map: XCTestCase, PBTestHelpers {
         assertDecodeSucceeds([10, 4, 8, 1, 16, 2]) {
             $0.mapInt32Int32 == [1: 2]
         }
+        // TODO: Why should this fail?  It appears to actually be valid.
```

Comment:
@tbkka I'll update this PR to disable this test for now, but do you recall the intent here and how were groups supposed to be handled with unknowns, I'm I reading this correctly as not handing correctly?